travel_platform/izi_travel_client.py

```diff
@@ -0,0 +1,306 @@
+import requests
+
+
+class BaseExceptionHandler(Exception):
+    """Base class for client exceptions."""
+    pass
+
+
+class IziTravelApiError(BaseExceptionHandler):
+    pass
+
+
+class IziTravelClient(object):
+
+    def __init__(self):
+        self.api_key = "9ba66b8d-2bdc-4674-bdc4-3cc67e2c25af"
+        self.api_url = "https://api.izi.travel/"
+        self.headers = {"X-IZI-API-KEY": self.api_key}
+        self.api_version = '1.2.4'
+        self.media_url = 'https://media.izi.travel/'
+        self.languages = "uk,en,ru"
+
+    def _check_response_status(self, response):
+        """
+        Raise an exception if status of a
+         response from IziTravel is not 200.
+        Args:
+            response (obj): Response object.
+        """
+        if response.status_code != 200:
+            raise IziTravelApiError("Status: {}. Response: {}.".format(
+                response.status_code, response.content))
+
+    def _prepare_image_url(self, content_provider_uuid,
+                           image_uuid, image_size='800x600'):
+        """
+        Args:
+            content_provider_uuid (str): Uuid of a content's provider.
+            image_uuid (str): Uuid of an image.
+            image_size (str): Image size. Available image sizes are:
+            1600 x1200, 800 x600, 480 x360, 240 x180, 120x90.
+        Returns:
+            str: Image url.
+        """
+        image_url = "{base_url}{provider}/{image}_{size}.jpg".format(
```

Comment:
You may just prepare a `dict` and use it for your string formatting.

Example:
```
data = {"name": "John", "age": 42}
"Hello, %(name)s. You are %(age)d old." % data
```

It's just a suggestion.